web-animations-1/Overview.bs

```diff
@@ -3366,6 +3369,8 @@ given <var>iteration progress</var>, <var ignore=''>current iteration</var> and
 1.  If <a>animation type</a> of the <var>target property</var> is
     <a>not animatable</a> abort this procedure
     since the effect cannot be applied.
+1.  If the [=effect target=] is `null` or cannot have computed property values
```

Comment:
Nit: Technically I guess this should say, "The [=keyframe effect=] does not have an [=effect target=]...` since we're talking about the model here. They fact that the lack of an associated effect target is represented by `null` in the programming interface is specific to the programming interface.